clang/lib/AST/ASTImporter.cpp

```diff
@@ -4579,7 +4579,12 @@ ExpectedDecl ASTNodeImporter::VisitVarDecl(VarDecl *D) {
     if (!RedeclOrErr)
       return RedeclOrErr.takeError();
   }
-
+  if (D->isInlineSpecified()) {
+    ToVar->setInlineSpecified();
+  }
+  if (D->isInline()) {
+    ToVar->setImplicitlyInline();
+  }
```

Comment:
I would like to move these lines to line 4513 (after `ToVar->setLexicalDeclContext(LexicalDC);`), and braces are not needed:
```
  if (D->isInlineSpecified())
    ToVar->setInlineSpecified();
  if (D->isInline())
    ToVar->setImplicitlyInline();
```

Reply:
Seems to be more reasonable. Code has been updated. Thanks!